docs/CHANGELOG.asciidoc

```diff
@@ -28,6 +28,18 @@
 
 //=== Regressions
 
+== {es} version 7.2.1
+
+=== Bug Fixes
+
+* Don't persist model state at the end of lookback if the lookback did not
+generate any input {ml-pull}527[#527] (issue: {ml-issue}519[#519])
+* Don't write model size stats when job is closed without any input
+{ml-pull}516[#512] (issue: {ml-issue}394[#394])
+* Fix an edge case causing spurious anomalies (false positives) if the variance
+in the count of events changed significantly throughout the period of a seasonal
+quantity. {ml-pull}489[#489]
```

Comment:
I think #489 was mislabelled.  It had the `v7.2.1` label but was only backported as far as the `7.3` branch.  I just fixed the labelling - sorry about that.

Reply:
Yes I just saw that. I think the original intention was that it would go into 7.2.1, but we decided to just release in 7.3. Looking at the PR I now wonder if the change log is wrong for 7.3. I'll raise a separate PR for that.

Reply:
Okay thanks, I've removed it from the 7.2.1 changelog